internal/resource/mappers/bimapper/bimapper.go

```diff
@@ -195,7 +271,8 @@ func (m *Mapper) MapLink(_ context.Context, _ controller.Runtime, res *pbresourc
 	return out, nil
 }
 
-func (m *Mapper) itemsByLink(link resource.ReferenceKey) []*pbresource.ID {
+func (m *Mapper) itemIDsByLink(link resource.ReferenceKey) []*pbresource.ID {
```

Comment:
Since `itemIDsByLink` and `itemRefsByLink` are so similar, it might be nice to extract a helper that returns the items for a link.